src/telegraf.ts

```diff
@@ -116,17 +117,28 @@ export class Telegraf<
 
   get webhookReply() {
     return this.telegram.webhookReply
-  } /* eslint brace-style: 0 */
+  }
 
   catch(handler: (err: any, ctx: TContext) => void) {
     this.handleError = handler
     return this
   }
 
   webhookCallback(path = '/') {
+    let mustFetchBotInfo = !this.botInfo
     return generateCallback(
       path,
-      (update: tt.Update, res: any) => this.handleUpdate(update, res),
+      async (update: tt.Update, res: http.ServerResponse) => {
+        if (mustFetchBotInfo) {
+          mustFetchBotInfo = false
+          try {
+            this.botInfo ??= await this.telegram.getMe()
+          } catch (err) {
+            debug('Could not initialize bot info', err)
```

Comment:
`handleUpdate` will run, without `botInfo`, if you don't rethrow

```suggestion
            debug('Could not initialize `botInfo`', err)
            throw err
```